src/extensions/mermaid.ts

```diff
@@ -17,6 +17,12 @@ export default (() => (tree: Root) => {
     }
     const htmlNode = node as unknown as Html
     htmlNode.type = 'html'
-    htmlNode.value = `<pre class="${CSS_CLASSNAME}">${node.value}</pre>`
+    const value = node.value
```

Comment:
Consider using a utility function or library to escape HTML characters instead of manually replacing each one. This would make the code more maintainable and less prone to errors. [important]